plugins/UFM_NDT_Plugin/ufm_sim_web_service/topo_diff/topo_diff.py

```diff
@@ -60,6 +60,24 @@ def __eq__(self, other):
     def __hash__(self):
         return self.unique_key.__hash__()
 
+def check_valid_peer_port_param(parameter_val):
+    '''
+    peer parameter value should not be none or empty string
+    :param parameter_val:
+    '''
+    if not parameter_val or parameter_val.isspace():
+        return False
+    return True
+
+def check_not_empty_valid_params(parameters_vals_list):
+    '''
+    peer parameters values should not be none or empty string
+    :param parameters_vals_list:
+    '''
+    for parameter_val in parameters_vals_list:
+        if not parameter_val or parameter_val.isspace():
```

Comment:
```suggestion
        if not check_valid_peer_port_param(parameter_val):
```

Reply:
no need to call function in this case as most of the times first condition will work

Reply:
there is the same code inside this function, so it will work the same way, but without code duplication